sys/dev/ath/if_ath_usb.c

```diff
@@ -59,28 +59,68 @@ __FBSDID("$FreeBSD$");
 #include <net80211/ieee80211_radiotap.h>
 #include <net80211/ieee80211_ratectl.h>
 
-#include "athreg.h"
-#include "ar5008reg.h"
-#include "athvar.h"
+// #include "athreg.h"
+// #include "ar5008reg.h"
```

Comment:
Not sure if it will be needed in the future or not. Probably reg values should not be used in USB module at all? I will remove these lines with other findings in this PR.

Reply:
Removed